src/Entity/Field.php

```diff
@@ -190,7 +196,7 @@ public function getValue(): ?array
      *
      * @return array|mixed|null
      */
-    public function getFieldValue()
+    public function getFlatenValue()
```

Comment:
Really like where this PR is going. But, this is a bit of an odd Method name.

Reply:
What do you propose? It returns single value if count($this->getValues()) === 1, array if it's >1 and null if it is zero.

Reply:
Then it should be `getFlatValue` or `getFlattenedValue`. "Flaten" looks like a typo. 😇